### DISCUSSION TITLE
Hability to store pre defined initial message

### DISCUSSION THREAD

**Discussion by Lunik** [langfuse/langfuse#2106] [Support] [answered]
Hi,

I'm looking for a way to store pre defined messages that my users can select to initiate a conversation with an AI Bot.
Same a something we can find in ChatGPT : 

![image](https://github.com/langfuse/langfuse/assets/12114079/7a327630-cf9e-4287-8270-f5e0d02da1ac)

Actually I'm using Langfuse Prompt feature to store the my prompt.
It could be useful if we could attach pre defined messages to that prompt (also with the compile feature).

Then when accessing and compiling the prompt with the SDK we could also get the pre defined messages.

**Comment by marcklingen** [comment, marked as answer]
ARe these pre-defined messages always the same? You could store them as an array within the `prompt.config` object to manage them together with your prompts.

**Reply by Lunik** [reply]
I will try with what you are proposing. It's promissing
